lib/init.js

```diff
@@ -7,7 +7,8 @@ const _ = require('lodash'),
   files = require('nymag-fs'),
   log = require('./services/log').withStandardPrefix(__dirname), // TODO: PASS IN LOGGER?
   acceptedIcons = ['icon.120x120.png', 'icon.180x180', 'icon.192x192.png'],
-  pageList = require('./page-list');
+  pageList = require('./page-list'),
+  usersList = require('./users-list');
```

Comment:
My understanding is that Amphora Search isn't a hard dependency of Amphora. Is there documentation for the API to which a search plug-in needs to conform?